google-cloud-network_connectivity-v1alpha1/CHANGELOG.md

```diff
@@ -1,2 +1,7 @@
 # Release History
 
+### 0.1.0 / 2021-01-25
+
+### 0.1.0 /  2021-01-25
```

Comment:
Please delete this line. I think you added it when writing the initial release notes? You don't need to do that. The tool creates the header for you.

Reply:
I have made the change.